notebooks/workflows/adviser-inspection-workflow.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
<p>"stable" should be uppercase</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thoth-station/notebooks/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>